functions.php

```diff
@@ -46,6 +46,7 @@ public function setup_hooks() {
 		add_filter( 'admin_post_thumbnail_html', array( $this, 'meta_wsm_banner_color' ), 10, 2 );
 		add_filter( 'pre_site_option_upload_filetypes', array( $this, 'set_upload_filetypes' ), 11, 1 );
 		add_filter( 'upload_mimes', array( $this, 'set_mime_types' ), 11, 1 );
+		add_filter( 'pre_get_posts', array( $this, 'web_extras_in_archives' ) );
```

Comment:
`pre_get_posts` is technically an action, though the filter should fire anyway because it all goes back to the same underlying code.